.github/workflows/build.yml

```diff
@@ -1,6 +1,7 @@
 name: build holochain
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
if we enable this the one runner that we've configured right now will be busy with both jobs :smile: i'm fine with trying it out and see how quick CI moves along